Fixtures/Miscellaneous/Plugins/MyBinaryToolPlugin/Binaries/MyVendedSourceGenBuildTool.artifactbundle/mytool-windows/mytool

```diff
@@ -0,0 +1,33 @@
+import argparse
```

Comment:
Could this be done in pure batch? This adds a python dependency to the tests, which might not be present on the host system.

Reply:
It can.  Could you please this with "request changes"